src/main/webapp/go.js

```diff
@@ -426,7 +434,54 @@ function checkUserDestinationGuess() { //eslint-disable-line
     return;
   }
 
-  // If userGuess is not an exact match, perform entity extraction.
+  // Check to see if userGuess can be used to identify the correct place
+  // using the Places library.
+  checkGuessWithPlaceIDs(userGuess);
+}
+
+/**
+ * Retrieves place ID for the answer, and compares this to the place
+ * ID retrieved for the user's guess.
+ * @param {String} userGuess User's guess for current destination.
+ */
+function checkGuessWithPlaceIDs(userGuess) {
+  const answerRequest = createRequestForPlaceID(hunt.getCurDestName());
```

Comment:
Place IDs for each destination are stored when the Destination object is created.  Instead of making the request to get it here, it would be simpler to just propagate the place ID from the Destination object to the HuntItem object.  Then you don't have to make this request because you already have the place ID.

Reply:
+1, but it's an optional suggestion as far as I'm concerned. Megan may feel more strongly.

I originally hesitated to suggest this because HuntItems currently don't contain the place ID, but thinking about it further, it shouldn't be too much work to start storing place IDs in them, and you don't have to worry about being backward compatible with old hunts in the datastore because people are always generating new hunts instead of using an existing ID.